CONTRIBUTING.md

```diff
@@ -62,63 +103,90 @@ to follow step 3 and beyond:
 5. Issue a Pull Request to the official repository
 6. Your Pull Request is reviewed by a committer and merged into the repository
 
-*Note*: While there are other ways to accomplish the steps using other tools, the examples here will assume the most
-actions will be performed via the `git` command line.
+**NOTE**: While there are other ways to accomplish the steps using other tools,
+the examples here will assume most actions will be performed via `git` on
+command line.
 
-For more information on maintaining a fork, please see the GitHub Help article [Fork a Repo](https://help.github.com/articles/fork-a-repo/) and information on
-[rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) from git.
+For more information on maintaining a fork, please see the GitHub Help article
+titled [Fork a Repo](https://help.github.com/articles/fork-a-repo/), and
+information on [rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing).
 
 ### Make Changes and Commit
 
-#### Before commit
+#### Before Commit
 
-At this point you have ready your changes, your new feature it's ready to be shipped, but, to avoid delays to merge, please be aware the build must past.
+Before committing, **you must ensure there are no linting errors and
+all tests pass.**
 
-Before commit, run the test command:
+To do this, run:
 
 ```bash
 yarn test
```

Comment:
To be more acurate, we should use `yarn test:all` that run all test included the e2e, it takes a bit more time but are the ones are run on CircleCi.